source/contracts/reporting/Market.sol

```diff
@@ -230,10 +228,8 @@ contract Market is DelegationTarget, ITyped, Initializable, Ownable, IMarket {
         // If the designated reporter showed up return the no show bond to the market creator. Otherwise it will be used as stake in the first report.
         if (_reporter == _initialReporter.getDesignatedReporter()) {
             require(_reputationToken.transfer(owner, _repBalance));
```

Comment:
Since there's only one transfer here now, this logic could be simplified, but i agree with what you've done here, changing as few lines as possible